ssmtp.conf

```diff
@@ -0,0 +1,4 @@
+FromLineOverride=YES
+mailhub=127.0.0.1
+UseTLS=NO
+UseSTARTTLS=NO
```

Comment:
nit: I like keeping trailing newlines at the EOF to keep git diffs clean ;P

Reply:
done!